### ISSUE TITLE
Right-click doesn't work

### ISSUE THREAD

**Issue by CecilFF4** [SonicTHI/SaveOurShip2Experimental#59]
Win 11 x64

Steam version. All DLC (though it doesn't matter if these are enabled or not). All files verified.

With this mod enabled, and no others (aside from dependencies), right-clicking does nothing. No context menu, no forced actions, no equipping weapons. Removing the mod fixes the issue.

**Comment by CecilFF4** [comment]
What I wrote for the version information was for Rimworld, not the mod.

I'm using [40d0d44](https://github.com/SonicTHI/SaveOurShip2Experimental/commit/40d0d4441107abc0fbb17b6fec3a8cf2b7beb22c) for the mod.

**Comment by SonicTHI** [comment]
Right, that s why it is required to post the [FULL LOG](https://github.com/SonicTHI/SaveOurShip2Experimental#troubleshooting-and-issue-reporting) but you skipped that part as well.
In any case i m not seeing this issue on my end.
